packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -786,6 +786,15 @@ abstract class MapsInitializerApi {
   @async
   PlatformRendererType initializeWithPreferredRenderer(
       PlatformRendererType? type);
+
+  /// Asks the Google Maps SDK to do the thread-blocking work it normally does
+  /// when a map is shown for the first time.
+  ///
+  /// This gives the developer the option to move that jank to a different
+  /// part of the map (typically, the app startup, where missed frames
+  /// aren't going to be noticed).
```

Comment:
This internal implementation API comment does not need to predict when people might choose to call this method.